spec/data-model/README.md

```diff
@@ -172,17 +179,13 @@ interface VariableRef {
 ```
 
 A `FunctionAnnotation` represents a _function_ _annotation_.
-In a `FunctionAnnotation`,
-the `kind` corresponds to the starting sigil of a _function_:
-`'open'` for `+`, `'close'` for `-`, and `'value'` for `:`.
-The `name` does not include this starting sigil.
+The `name` does not include the `:` starting sigil.
```

Comment:
I think there's a problem here. We're using the word `name` (here and in the interface definitions above and below). This is fine for `VariableRef`, but not for `FunctionAnnotation`, which should use `identifier` (that is, including the namespace)

Reply:
I've spun this off into a separate issue #575, as it's a separate concern from the markup implementation.